sync.yml

```diff
@@ -23,7 +23,7 @@
   tags: permissions
 
   vars:
-    skip_permission_check: false
+    skip_permission_check: true
```

Comment:
What does this change and what are the implications of that?

Reply:
it just skip the checking of permissions below by default. The permissions are set explicitly at the end of the `install.yml` play, so I don't think this step is necessary.

if permissions turn out to be a problem in practice down the line, we can consider doing this differently

Reply:
I have now changed this behavior so that the permission check is tagged and can be skipped with `--skip-tags` if needed